Build failure on darwin arm64 ("Apple Silicon")

Hello,

This package does not compile on the new darwin+arm64 (aka "Apple Silicon") port of Go (new in 1.16).

```bash
$ GOOS=darwin GOARCH=arm64 go build
# github.com/albenik/go-serial/v2
./serial_unix.go:298:13: p.retrieveTermSettings undefined (type *Port has no field or method retrieveTermSettings)
./serial_unix.go:315:12: p.applyTermSettings undefined (type *Port has no field or method applyTermSettings)
./serial_unix.go:401:13: p.retrieveTermSettings undefined (type *Port has no field or method retrieveTermSettings)
./serial_unix.go:422:10: p.applyTermSettings undefined (type *Port has no field or method applyTermSettings)
./termios_unix.go:20:23: undefined: tcCMSPAR
./termios_unix.go:25:23: undefined: tcCMSPAR
./termios_unix.go:30:23: undefined: tcCMSPAR
./termios_unix.go:33:6: undefined: tcCMSPAR
./termios_unix.go:38:22: undefined: tcCMSPAR
./termios_unix.go:41:6: undefined: tcCMSPAR
./termios_unix.go:41:6: too many errors
```

--- a/constants_darwin_64.go
+++ b/constants_darwin_64.go
@@ -4,6 +4,9 @@
 // that can be found in the LICENSE file.
 //
 
+// +build darwin
+// +build amd64 arm64
+
 package serial
 
 import "golang.org/x/sys/unix"

